include/sys/arc_impl.h

```diff
@@ -964,6 +964,13 @@ typedef struct arc_evict_waiter {
 #define	arc_c_max	ARCSTAT(arcstat_c_max)	/* max target cache size */
 #define	arc_sys_free	ARCSTAT(arcstat_sys_free) /* target system free bytes */
 
+#define	arc_anon	(&ARC_anon)
+#define	arc_mru		(&ARC_mru)
+#define	arc_mru_ghost	(&ARC_mru_ghost)
```

Comment:
I think this style of macros make the code harder to read.  Since they are already all over the ARC, I'm OK with the change as-is, but if you or anyone else would like to improve the code, it would be great to either:

- change all the users to do `&ARC_anon`, etc. instead of `arc_anon`

or

- make these inline functions, like `inline arc_state_t* arc_anon() {return (&ARC_anon);)`